tests/API_smoke_test.postman_collection.json

```diff
@@ -464,6 +465,3139 @@
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\"bgTarget\":{\"high\":10,\"low\":3.9},\"units\":{\"bg\":\"mmol/L\"}}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{api}}/metadata/{{userId}}/settings",
+									"host": [
+										"{{api}}"
+									],
+									"path": [
+										"metadata",
+										"{{userId}}",
+										"settings"
+									]
+								},
+								"description": "Changing BG units to mmol/L, another aspect a user can change in the user profile."
+							},
+							"response": []
+						},
+						{
+							"name": "Retrieve settings",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "bef1356d-82ff-4212-9b5a-d6f6c41afd02",
+										"exec": [
+											"const assert = eval(globals.loadAsserts);",
+											"assert.common(200, 500, false, pm.response.json(),\"userSettingsSchema\");",
+											"",
+											"pm.test(\"Verify settings change to mmol\", function () {",
+											"    var jsonData = pm.response.json();",
+											"    pm.expect(jsonData.units.bg).to.eql(\"mmol/L\");",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "x-tidepool-session-token",
+										"type": "text",
+										"value": "{{sessionToken}}"
+									}
+								],
+								"url": {
+									"raw": "{{api}}/metadata/{{userId}}/settings",
+									"host": [
+										"{{api}}"
+									],
+									"path": [
+										"metadata",
+										"{{userId}}",
+										"settings"
+									]
+								},
+								"description": "Verifying the change to mmol/L was stored."
+							},
+							"response": []
+						},
+						{
+							"name": "Tidepool data donation opt-in",
+							"event": [
+								{
+									"listen": "prerequest",
+									"script": {
+										"id": "146fda7e-6cd2-47e1-8aa0-38e678d68fc0",
+										"exec": [
+											""
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "test",
+									"script": {
+										"id": "10167115-0928-4636-8932-0296b3ad66b5",
+										"exec": [
+											"const assert = eval(globals.loadAsserts);",
+											"assert.common(200, 500, false, pm.response.json(),\"careteamInviteSchema\");"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "x-tidepool-session-token",
+										"value": "{{sessionToken}}",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\"email\":\"bigdata@tidepool.org\",\"permissions\":{\"view\":{},\"note\":{}}}",
```

Comment:
q: email as a variable or does it not matter?? I have no opinion either way just wanted to note it